.travis.yml

```diff
@@ -11,3 +11,7 @@ script:
   - ./gradlew check -x signArchives -x java-it-tests:common:check -x java-it-tests:driver20:check -x java-it-tests:driver21:check -x java-it-tests:driver30:check
   - jdk_switcher use oraclejdk8
   - ./gradlew java-it-tests:common:check java-it-tests:driver20:check java-it-tests:driver21:check java-it-tests:driver30:check -x signArchives
+
+cache:
+  directories:
+  - "$HOME/.gradle/caches"
```

Comment:
It seems to shave some time off the build which is nice.  It is probably is a net positive for the travis infra as well as they don't have to go out and fetch the deps every time too.